posthog/models/filters/mixins/common.py

```diff
@@ -321,3 +321,13 @@ def target_entity_type(self) -> Optional[str]:
     @include_dict
     def entity_type_to_dict(self):
         return {"entity_type": self.target_entity_type} if self.target_entity_type else {}
+
+
+class WindowMixin(BaseParamMixin):
```

Comment:
this looks like a copy of the above filter param. did you mean for this to be different/used differently? Otherwise, can be removed